pinot-spi/src/main/java/org/apache/pinot/spi/exception/QueryErrorCode.java

```diff
@@ -57,6 +57,7 @@ public enum QueryErrorCode {
   UNKNOWN_COLUMN(710, "UnknownColumnError"),
   ///  Error while planning the query. For example, trying to run a colocated join on non-colocated tables.
   QUERY_PLANNING(720, "QueryPlanningError"),
+  ///  Query already errored out.
```

Comment:
Does this javadoc applies to UNKNOWN or you forgot to remove the javadoc of the removed literal?

Reply:
You guessed it, forgot to remove it as well. Sorry about that.